cmd/keytransparency-server/frontend.go

```diff
@@ -188,7 +189,11 @@ func main() {
 	if err != nil {
 		log.Fatalf("Failed to create committer: %v", err)
 	}
-	queue := queue.New(context.Background(), etcdCli, *mapID, factory)
+	mutations, err := mutations.New(sqldb, *mapID)
+	if err != nil {
+		log.Fatalf("Failed to create mutations object: %v", err)
+	}
+	queue := queue.New(context.Background(), etcdCli, *mapID, factory, mutations)
```

Comment:
Does giving `queue` access to `mutations` invite scope creep? 
Would it be better to have `keyserver` write to `mutations` directly?